### DISCUSSION TITLE
Side-bar toggle functionality

### DISCUSSION THREAD

**Discussion by rahulAnand1999** [shadcn-ui/ui#3684] [Blocks Request]
### Description

Please provide a template where we have a sidebar which has toggle functionality..
Initially it should show the full side bar,,on clicking the toggle button the width of the side bar should descrese and only icons should be visible in the side bar instead of full menu

**Comment by gnanadeep-t** [comment]
Check my repo
https://github.com/gnanadeep-t/Collapsible-sidebar.git

**Reply by mikdanjey** [reply]
Please deploy & give the demo link
